src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -65,6 +64,10 @@ afterEach(() => {
 });
 
 describeEachAppLayout({ sizes: ['desktop'] }, ({ theme }) => {
```

Comment:
upon further investigation, the useMobile hook is not returning true inside the tests as it does on the local device, therefor the mobile-bar classes are not being applied at the time of [assertion found here.](https://github.com/cloudscape-design/components/blob/main/src/app-layout/__tests__/utils.tsx#L88)

This assertion works for many other unit tests, so need to determine why the mock is not working on this test specifically.  This will be done on a separate PR